src/org/rascalmpl/library/util/PathConfig.java

```diff
@@ -426,6 +426,20 @@ public PathConfig parse(String pathConfigString) throws IOException {
      * Configure paths for the rascal project itself, so if someone has rascal open in their IDE for example, or is starting a REPL for rascal
      */
     private static void buildRascalConfig(ISourceLocation workspaceRascal, RascalConfigMode mode, IList mavenClassPath, IListWriter srcs, IListWriter libs, IListWriter messages) throws IOException {
+        // we want to help rascal devs work on rascal files without having to always open a RascalShell.
+        // note: we except that if you do changes in Java, you won't see them and might get import errors
```

Comment:
import errors -> "ClassNotFoundException", "MethodNotFoundException", "ClassInitializationError" good to mention explicitly here.